force-app/main/default/classes/MathCalculatorTest.cls

```diff
@@ -0,0 +1,18 @@
+@isTest
+public with sharing class MathCalculatorTest {
```

Comment:
This test is not sufficient, please add additional test methods to have good amount of coverage